src/Microsoft.Owin.Security.OpenIdConnect/OpenidConnectAuthenticationHandler.cs

```diff
@@ -150,6 +150,18 @@ private string CurrentUri
                     Scope = Options.Scope,
                     State = OpenIdConnectAuthenticationDefaults.AuthenticationPropertiesKey + "=" + Uri.EscapeDataString(Options.StateDataFormat.Protect(properties)),
                 };
+                if (properties.Dictionary.ContainsKey("display"))
```

Comment:
Do we have tests for this somewhere?

Reply:
No, I didn't change any of the tests to poke at these added properties.  Frankly, I'm a newb and I'm not sure where (in the test projects) such code should go.  Under "tests/Microsoft.Owin.Security.Tests" I don't see a folder for OpenID Connect.  I see tests for Google, OAuth, Facebook, etc.  

I'm probably just looking in all the wrong places.